localstack/services/cloudformation/resource_providers/aws_cloudformation_stack.py

```diff
@@ -0,0 +1,198 @@
+                "CAPABILITY_AUTO_EXPAND",
+            ]
+
+            create_params["Parameters"] = [
+                {
+                    "ParameterKey": k,
+                    "ParameterValue": str(v).lower() if isinstance(v, bool) else str(v),
+                }
+                for k, v in create_params.get("Parameters", {}).items()
+            ]
+
+            result = request.aws_client_factory.cloudformation.create_stack(**create_params)
+            model["Id"] = result["StackId"]
+
+            request.custom_context[REPEATED_INVOCATION] = True
+            return ProgressEvent(
+                status=OperationStatus.IN_PROGRESS,
+                resource_model=model,
+                custom_context=request.custom_context,
+            )
+
+        descriptions = request.aws_client_factory.cloudformation.describe_stacks(
+            StackName=model["Id"]
+        )
+        if not descriptions.get("Stacks"):
+            raise Exception("Stack not found")
+
+        stack = descriptions["Stacks"][0]
+        if stack["StackStatus"] == "CREATE_COMPLETE":
+            model["Outputs"] = {o["OutputKey"]: o["OutputValue"] for o in stack.get("Outputs", [])}
+            return ProgressEvent(
+                status=OperationStatus.SUCCESS,
+                resource_model=model,
+                custom_context=request.custom_context,
+            )
+        if stack["StackStatus"] == "CREATE_FAILED":
+            return ProgressEvent(
+                status=OperationStatus.FAILED,
+                resource_model=model,
+                custom_context=request.custom_context,
+            )
+        return ProgressEvent(
+            status=OperationStatus.IN_PROGRESS,
+            resource_model=model,
+            custom_context=request.custom_context,
+        )
+
+    def read(
+        self,
+        request: ResourceRequest[CloudFormationStackProperties],
+    ) -> ProgressEvent[CloudFormationStackProperties]:
+        """
+        Fetch resource information
+
+
+        """
+        raise NotImplementedError
+
+    def delete(
+        self,
+        request: ResourceRequest[CloudFormationStackProperties],
+    ) -> ProgressEvent[CloudFormationStackProperties]:
+        """
+        Delete a resource
+
+
+        """
+
+        model = request.desired_state
+        if not request.custom_context.get(REPEATED_INVOCATION):
+            request.aws_client_factory.cloudformation.delete_stack(StackName=model["Id"])
+
+            request.custom_context[REPEATED_INVOCATION] = True
+            return ProgressEvent(
+                status=OperationStatus.IN_PROGRESS,
+                resource_model=model,
+                custom_context=request.custom_context,
+            )
+
+        descriptions = request.aws_client_factory.cloudformation.describe_stacks(
+            StackName=model["Id"]
+        )
+
+        if not descriptions.get("Stacks"):
+            raise Exception("Stack not found")
```

Comment:
same as above.

The delete operation should also be idempotent, i.e. a not found exception should still lead to a success.